packages/http/src/util.js

```diff
@@ -76,24 +76,27 @@ export function request(method, path, params, callback = s => s) {
       key => key.toLowerCase() === 'content-type'
     );
 
-    const hasContentType = !!contentTypeKey;
-    const contentTypeValue = hasContentType ? headers[contentTypeKey] : '';
+    const hasContentType = contentTypeKey && headers[contentTypeKey];
 
-    if (hasContentType) {
-      headers = { ...headers };
-    } else if (contentType === 'form') {
+    const contentTypeValue = hasContentType
```

Comment:
This ternary is a little strange. I'm always suspicious of ternary expressions where the `false` branch is a falsy value. They are very rarely needed. It makes me think something isn't quite right about the solution.